src/main/java/io/pakland/mdas/githubstats/infrastructure/shell/validation/EntityTypeValidator.java

```diff
@@ -0,0 +1,22 @@
+package io.pakland.mdas.githubstats.infrastructure.shell.validation;
+
+import io.pakland.mdas.githubstats.domain.EntityType;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class EntityTypeValidator implements InputValidator<String> {
```

Comment:
If we allow the user to have 3 command options (`user`, `team` and `organization`) we are already validating their input (using spring shell). With this, we are adding extra complexity to the code.  
Another UI thing, small but to take into account, is the fact that spring shell allows us to generate documentation for the command options we provide. Using the approach followed in this pr, the documentation is non-existent (there's no documentation in the `ShellConfiguration` file).